cibuildwheel/macos.py

```diff
@@ -11,7 +11,7 @@
 from .environment import ParsedEnvironment
 from .util import (BuildOptions, BuildSelector, NonPlatformWheelError, download,
                    get_build_verbosity_extra_flags, get_pip_script,
-                   prepare_command)
+                   prepare_command, instal_certifi_script)
```

Comment:
Typo in this import "instal" versus "install"

Reply:
Ah nevermind. That's how it is in the other module too.

Reply:
Still a typo, though ;-)

Reply:
fixed

Reply:
Great, thanks! :-)